src/components/query/QuerySummary.vue

```diff
@@ -117,15 +117,21 @@
         </dd>
       </dl>
     </el-row>
+    <el-row v-if="query.run.warnings && query.run.warnings.length > 0">
+      <el-collapse>
+        <run-warnings :warnings="query.run.warnings"></run-warnings>
+      </el-collapse>
+    </el-row>
     <el-row v-if="query.isCompleted()">
-      <el-divider></el-divider>
+      <el-divider v-if="!(query.run.warnings && query.run.warnings.length > 0)"></el-divider>
```

Comment:
Why?

Reply:
To avoid having a double divider. The warning component already put one in the bottom.